src/main/resources/com/microfocus/application/automation/tools/pipelineSteps/SseBuildAndPublishStep/config.jelly

```diff
@@ -69,23 +69,15 @@
         <f:entry title="Domain" field="almDomain">
             <f:textbox name="sse.almDomain" value="${instance.sseModel.almDomain}"/>
         </f:entry>
-        
-        <f:entry>
```

Comment:
Does it mean that proxy is no longer supported in SSE?

Reply:
No. SSE will use Jenkins system proxy settings.